addons/stock_dropshipping/models/purchase.py

```diff
@@ -31,6 +31,13 @@ def _prepare_stock_moves(self, picking):
         res = super(PurchaseOrderLine, self)._prepare_stock_moves(picking)
         for re in res:
             re['sale_line_id'] = self.sale_line_id.id
+            if self.sale_line_id:
```

Comment:
I think that what defines a dropshipping is not to have a sales order line, but to have a destination address (which can be put manually):

```suggestion
            if self.order_id.dest_address_id:
```